Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/runtime/DateTime.java

```diff
@@ -296,10 +294,8 @@ public Date toJavaDate() {
         return java.util.Date.from(dateTime.toInstant());
     }
 
-    public static DateTime fromJavaDate(Date date) {
```

Comment:
This is a very useful function that is used in other projects. I would like to know why it has been removed.

Reply:
The IDE said it was unused and it's my instinct to clean up dead code.   

Feel free to add it back.

Reply:
I did a quick review of cqf-tooling and clinical-reasoning, this function is used when parsing the input reporting period. So it's only used one place I'm fine either way.